### PR TITLE
Docs: Add upgrade/breaking note about SQL data sources and data frames

### PR DESCRIPTION
**Pull request by marefr**
**What this PR does / why we need it**:
Add upgrade/breaking note about SQL data sources and data frames.

**Which issue(s) this PR fixes**:
Ref #35534 

**Special notes for your reviewer**:

### CODE REVIEW

**Review by torkelo — APPROVED** [review]

File: docs/sources/installation/upgrading.md [hunk 1]
```diff
@@ -326,3 +326,11 @@ For example, if you want an alert to be `INFO`-level in VictorOps, create a tag
...
+
+### Postgres, MySQL, Microsoft SQL Server data sources
+
+Grafana v8.0 changes the underlying data structure to [data frames]({{< relref "../developers/plugins/data-frames.md" >}}) for the Postgres, MySQL, Microsoft SQL Server data sources. This has the implication that  any _Time series_ query result is returned in a [wide format]({{< relref "../developers/plugins/data-frames.md#wide-format" >}}) and that you might have to do some manual migration to make visualizations work as it did before.
+
+For any existing panels/visualizations using a _Time series_ query where the time column is only needed for filtering the time range, e.g. using the bar gauge or pie chart panel, it's suggested to use a _Table query_ instead and not including the time column as a field in the response.
+
+For further detailed instructions and workarounds see this [issue comment](https://github.com/grafana/grafana/issues/35534#issuecomment-861519658).
```
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:336]:
```suggestion
Grafana v8.0 changes the underlying data structure to [data frames]({{< relref "../developers/plugins/data-frames.md" >}}) for the Postgres, MySQL, Microsoft SQL Server data sources. As a result, a _Time series_ query result gets returned in a [wide format]({{< relref "../developers/plugins/data-frames.md#wide-format" >}}). To make the visualizations work as they did before, you might have to do some manual migrations.

		For any existing panels/visualizations using a _Time series_ query, where the time column is only needed for filtering the time range, for example, using the bar gauge or pie chart panel, we recommend that you use a _Table query_ instead and exclude the time column as a field in the response.
		
		Refer to this [issue comment](https://github.com/grafana/grafana/issues/35534#issuecomment-861519658) for detailed instructions and workarounds.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc change suggestions.

**Review by achatterjee-grafana — APPROVED** [review]

File: docs/sources/installation/upgrading.md [hunk 2]
```diff
@@ -326,3 +326,11 @@ For example, if you want an alert to be `INFO`-level in VictorOps, create a tag
...
+
+### Postgres, MySQL, Microsoft SQL Server data sources
+
+Grafana v8.0 changes the underlying data structure to [data frames]({{< relref "../developers/plugins/data-frames.md" >}}) for the Postgres, MySQL, Microsoft SQL Server data sources. As a result, a _Time series_ query result gets returned in a [wide format]({{< relref "../developers/plugins/data-frames.md#wide-format" >}}). To make the visualizations work as they did before, you might have to do some manual migrations.
+
+		For any existing panels/visualizations using a _Time series_ query, where the time column is only needed for filtering the time range, for example, using the bar gauge or pie chart panel, we recommend that you use a _Table query_ instead and exclude the time column as a field in the response.
```
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:334]:
```suggestion
For any existing panels/visualizations using a _Time series_ query, where the time column is only needed for filtering the time range, for example, using the bar gauge or pie chart panel, we recommend that you use a _Table query_ instead and exclude the time column as a field in the response.
```

File: docs/sources/installation/upgrading.md [hunk 3]
```diff
@@ -326,3 +326,11 @@ For example, if you want an alert to be `INFO`-level in VictorOps, create a tag
...
+		
+		Refer to this [issue comment](https://github.com/grafana/grafana/issues/35534#issuecomment-861519658) for detailed instructions and workarounds.
```
Comment (achatterjee-grafana) [inline docs/sources/installation/upgrading.md:336]:
```suggestion
Refer to this [issue comment](https://github.com/grafana/grafana/issues/35534#issuecomment-861519658) for detailed instructions and workarounds.
```

**Pull request merged**